<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPresetManagerDlg</class>
 <widget class="QWidget" name="DlgPresetManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_results">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>811</width>
     <height>571</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Midi controller name or manufacturers&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_local">
    <attribute name="title">
     <string>Local</string>
    </attribute>
    <widget class="QPushButton" name="btn_localleft">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_localright">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>210</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidgetLocal">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>731</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_cloud">
    <attribute name="title">
     <string>Cloud</string>
    </attribute>
    <widget class="QPushButton" name="btn_cloudleft">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cloudright">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>210</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidgetCloud">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>731</width>
       <height>511</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>811</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_search">
    <item>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>38</height>
       </size>
      </property>
      <property name="toolTip">
       <string>midi controller name or manufactuers</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_search">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>38</height>
       </size>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../../.designer/backup/1367219166_23273.ico</normaloff>../../../../../../.designer/backup/1367219166_23273.ico</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_statisticalresult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>About 30+ presets, 0.5 second used </string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>811</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_bottom">
    <item>
     <widget class="QLabel" name="label_notification">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You have choiced &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;American Audio RADIUS 2000 CH1&lt;/span&gt; preset, wait for more operation...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_apply">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>38</height>
       </size>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../../.designer/backup/1367310629_reload_refresh.png</normaloff>../../../../../../.designer/backup/1367310629_reload_refresh.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
